### PR TITLE
Add missing docs for copy_fields and truncate_fields

### LINKED ISSUE #12633: Missing docs for copy_fields and truncate_fields processors
There is no documentation [listed](https://www.elastic.co/guide/en/beats/filebeat/7.2/defining-processors.html#processors) for the [`copy_fields`](https://github.com/elastic/beats/pull/11303) and [`truncate_fields`](https://github.com/elastic/beats/pull/11297) processors.

### PR DESCRIPTION
**Pull request by kvch**
I added documentation for the processors `copy_fields` and `truncate_fields`.

Closes #12633

### CODE REVIEW

**Comment by dedemorton** [comment]
Thanks for contributing! As I mentioned in chat, we are moving the processor docs to live under the code modules, so it would be better to wait until the following PR is merged: https://github.com/elastic/beats/pull/14410 then break your additions into separate files.

Getting the docs to build from the processors/*/docs directories will take a little extra work (we're working on that). I appreciate your patience.

**Comment by kvch** [comment]
Ok. I added the label "in progress" as this PR requires more work.

**Comment by dedemorton** [comment]
OK, I've merged my PR.  You'll need to move the new content to separate files and put them under processors/*/docs then update libbeat/docs/processors-list.asciidoc to pull the files into the build.

Let me know if you need any help.

**Comment by kvch** [comment]
@dedemorton I updated the PR. Could you please review it?

File: libbeat/processors/actions/docs/truncate_fields.asciidoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[[truncate-fields]]
+=== Truncate fields
+
+The `truncate_fields` truncates a field to a given size if bigger. If the size of the field is smaller than
+the limit, the field is left as is.
+
+`fields`:: List of `from` and `to` pairs to copy from and to.
+`max_bytes`:: Maximum number of bytes in a field. Mutually exclusive with `max_characters`.
+`max_characters`:: Maximum number of characters in a field. Mutually exclusive with `max_bytes`.
+`fail_on_error`:: (Optional) If set to true, in case of an error the changes to
```
Comment (dedemorton) [inline libbeat/processors/actions/docs/truncate_fields.asciidoc]: What you have here is consistent with how we describe this field elsewhere. I think the description needs to be improved, but I'll do that in a follow-on PR so I can apply the change to the docs for other processors.

Adding needs_edit label to remind me.

File: libbeat/processors/actions/docs/truncate_fields.asciidoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+the event are reverted, and the original event is returned. If set to `false`,
+processing continues also if an error happens. Default is `true`.
+`ignore_missing`:: (Optional) Whether to ignore events which lack the source
```
Comment (dedemorton) [inline libbeat/processors/actions/docs/truncate_fields.asciidoc]:
```suggestion
`ignore_missing`:: (Optional) Whether to ignore events that lack the source
```

**Comment by dedemorton** [comment]
I've created a meta issue to track the content that requires an edit, so I'm removing the needs_edit
label.

**Pull request merged**